lib/screens/common/common_functions.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 SEVA AUTHORS. All Rights Reserved.
+//
+// (change the version and the date whenver anyone worked upon this file)
+// Version-0.5.2
+// Date-{02-10-2020}
+///
+/// @fileoverview common functions file : common functions used at multiple
+///  files.
+///
+
+import 'package:mvp/models/newCart.dart';
+import 'package:mvp/models/storeProducts.dart';
+
+class helperFunctions {
+  static void helper(
```

Comment:
how can we make this generic.? What I mean is, currently we have hard-coded all the `quantity metrics`, we should make it dynamic ? We can have a generic class ?